src/radius/radius.h

```diff
@@ -6,21 +6,11 @@
  * See README for more details.
  */
 
-/**
- * @file radius.h
- * @authors Jouni Malinen, Alexandru Mereacre
- * @brief RADIUS message processing.
```

Comment:
Same here, please add info about the upstream source of these changes (e.g. commit hash and git URL).

Reply:
Ok.